build_tools/travis/script.sh

```diff
@@ -20,6 +20,10 @@ if [[ "$ENABLE_PACKAGING" == "1" ]]; then
     # Remove conda used from wrapping
     rm -rf ${HOME}/miniconda_wrap
 
+    if [[ "$BLASLAPACK_IMPLEMENTATION" == "MKL" ]]; then
+        sudo rm -rf /opt/intel
+    fi
```

Comment:
in this script replace:

/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall)"

by:

```
curl -O https://raw.githubusercontent.com/Homebrew/install/master/uninstall
chmod +x 
uninstall
./uninstall --force
```

that's why travis fails on mac